### PR TITLE
Added text.

### PR DESCRIPTION
**Pull request by MikeCamel**
My first Rust!

### CODE REVIEW

File: payload/src/main.rs [hunk 1]
```diff
@@ -28,4 +28,7 @@ fn main() {
     for aux in &mut reader {
         println!("aux: {:?}", aux);
     }
+
+    println!();
+    println!("that's a nice TNETENNBA!");
```
Comment (npmccallum) [inline payload/src/main.rs:33]:
```suggestion
    println!("Good morning, that's a nice tnetennba!");
```

Comment (MikeCamel) [reply payload/src/main.rs:33]: The capitalisation is disputed: I'll amend slightly.  The salutation should, indeed, be included.

File: payload/src/main.rs [hunk 2]
```diff
@@ -28,4 +28,7 @@ fn main() {
     for aux in &mut reader {
         println!("aux: {:?}", aux);
     }
+
+    println!();
+    println!("Good morning, that's a nice Tnetennba!");
```
Comment (lkatalin) [inline payload/src/main.rs:33]:
```suggestion
    println!("\nGood morning, that's a nice Tnetennba!");
```

Comment (MikeCamel) [reply payload/src/main.rs:33]: Is that considered good practice these days?  Fewer lines, but less legible.  Rejected!

Comment (npmccallum) [reply payload/src/main.rs:33]: `println!()` is preferred.

Technically, `\n` as newline character isn't portable. Though in practice it works pretty much everywhere except Windows.

Comment (lkatalin) [reply payload/src/main.rs:33]: I had no idea! Learned something new.

**Comment by MikeCamel** [comment]
1. I've squashed the commits.  Do I need to repull?
2. Good idea.

**Comment by MikeCamel** [comment]
>     1. Please squash the commits. We only want a single commit for this.
> 
>     2. Also, I'm wondering if this might be a better fit for a C-language example. We can demo the Rust payload. But then also show that C or C++ works just as well.

2. good idea.

**Comment by npmccallum** [comment]
@MikeCamel Let's pause on this PR until @haraldh gets `musl-gcc -static-pie` working. Then we'll see if we can't come up with a C-lang demo.

**Comment by npmccallum** [comment]
The demo is done and this PR isn't needed anymore. But we did use your text. :)

**Pull request closed without merging**